app/enssubdomain/models.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models
+from django.utils import timezone
+from django.utils.timezone import localtime
+
+# Create your models here.
+from economy.models import SuperModel
+
+
+def get_time():
+    """Get the local time."""
+    return localtime(timezone.now())
+
+
+class ENSSubdomainRegistration(SuperModel):
+    """Define the schema for storing ENS sub-domain registration info"""
+
+    github_handle = models.CharField(max_length=255, blank=False, db_index=True)
```

Comment:
You can drop `blank=False` here - Defaults to false.
On second thought... do you mind switching this `github_handle` to `profile = models.ForeignKey('dashboard.Profile', related_name='ens_registration', null=True, on_delete=models.SET_NULL)`

Then you can do: `<ENSSubdomainRegistration_obj>.profile.username`